### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Rmadeye**
-added docs to functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+### **Usage** ###
+
+```bash
+python predict.py --cf_results DIR --use_pairwise BOOL --save_csv BOOL
+
+ ```
+ | Argument        | Description |
+|:-------------:|-------------|
+| **`--cf_results`** | Colabfold output directory with saved embeddings via --save-representations option |
+| **`--use_pairwise`** | Flag for using sorting pair representations
```
Comment (jludwiczak) [inline README.md]: Rephrase `Flag for using sorting pair representations`. Since this option is used by default maybe it can be omitted?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+| **`--save_csv`** | Flag for saving output to csv file
+
+```bash
+python predict.py --cf_results tests/data/0 --use_pairwise True --save_csv False
```
Comment (jludwiczak) [inline README.md]: Bool flags can be set in `argparse` just using `--use_pairwise` instead of `--use_pairwise True`. Perhaps remove this if agreed to comment above. `--save_csv` can take a filename option.

File: analyze.ipynb
Comment (jludwiczak) [inline analyze.ipynb:1]: Rather not needed in main.

File: tests/data/0/3e7k.a3m
Comment (jludwiczak) [inline tests/data/0/3e7k.a3m:1]: Remove this and other raw CF outputs that are not numpy matrices.

File: tests/test_dc2.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import sys
+import os
+import glob
+import pandas as pd
+import numpy as np
+
+root_directory = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+
+sys.path.append(root_directory)
```
Comment (jludwiczak) [inline tests/test_dc2.py]: Still not addressed.

File: src/train.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+# from sklearn.model_selection import KFold, train_test_split
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score, f1_score
+from sklearn.preprocessing import LabelEncoder, StandardScaler
+
+CALC_PATHS = ''
+
+
+def get_x(id_: int, rank: int, model: str = "af2",
```
Comment (jludwiczak) [inline src/train.py]: Are both get_x or get_af2_emb needed?

File: src/predictor.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    y_pred_bin = avg_proba.argmax(axis=1)
+
+    data = {avg_proba[0][0]: std_proba[0][0], avg_proba[0][1]: std_proba[0][1], avg_proba[0][2]: std_proba[0][2]}
+    data = {'prob_dimer':avg_proba[:,0],
+            'prob_dimer_std':std_proba[:,0],
+             'prob_trimer':avg_proba[:,1],
+                'prob_trimer_std':std_proba[:,1],
+                'prob_tetramer':avg_proba[:,2],
+                'prob_tetramer_std':std_proba[:,2],
+                'y_pred':y_pred_bin[0],}
```
Comment (jludwiczak) [inline src/predictor.py]: Indentation.

**Review by jludwiczak — CHANGES_REQUESTED** [review]

File: src/train.py (hunk 4 above)
Comment (Rmadeye) [reply src/train.py]: nope

**Review by jludwiczak — APPROVED** [review]
